include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -189,6 +363,19 @@ namespace LinearAlgebra
        * improve performance.
        */
       void
+      import_elements(
+        const ReadWriteVector<Number> &V,
+        VectorOperation::values        operation,
+        const Teuchos::RCP<const Utilities::MPI::CommunicationPatternBase>
+          &communication_pattern = {});
```

Comment:
We now have two overloads that only differ in the defaulted argument. Thus, every call omitting the second parameter becomes ambiguous. Would you mind removing the default parameter and create a third overload that only takes one argument to resolve that issue?

Reply:
done!